Index: Model/Build/MakeAll.sh
===================================================================
--- Model/Build/MakeAll.sh	(revision 3586)
+++ Model/Build/MakeAll.sh	(working copy)
@@ -7,7 +7,10 @@
 ulimit -s unlimited
 rm -f $APSIM/Model/Build/Build.out
 
-if [ "$1" != "clean" ]; then tclsh VersionStamper.tcl; fi
+if [ "$1" != "clean" ]; then 
+   gmcs VersionStamper.cs
+   mono ./VersionStamper.exe Directory=$PWD
+fi
 
 # -------------------------------------------------------------
 # This script compiles the APSIM engine and modules
@@ -114,4 +117,4 @@
 ./MakeProject.sh FarmSimGraze     $1
 ./MakeProject.sh ApsimX           $1
 ./MakeProject.sh JobScheduler/CreatePatch      $1
-
+./MakeProject.sh ApsimModel       $1
